### PR TITLE
Support multiple sources

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/ACM-5868

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@
 .build-harness-bootstrap
 build-harness
 build-harness-extensions
+.vscode/settings.json
```
Comment (chenz4027) [inline .gitignore:16]: Local settings for jest configuration.

File: frontend/src/wizards/Argo/schema.json [hunk 2]
```diff
@@ -61,7 +53,7 @@
                       }
                     }
                   },
-                  "required": ["project", "source", "destination"]
+                  "required": ["project", "destination"]
```
Comment (chenz4027) [inline frontend/src/wizards/Argo/schema.json:56]: Neither source or sources is required as appset can has either of them.

File: frontend/src/wizards/Argo/MultipleSourcesSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+            <Label style={{ marginRight: 10 }} color="grey">
+              {t('Git')}
+            </Label>
+          </WizHidden>
+          <WizHidden hidden={(data) => data.repositoryType !== 'helm'}>
+            <Label style={{ marginRight: 10 }} color="grey">
+              {t('Helm')}
+            </Label>
+          </WizHidden>
+          <WizTextDetail path="repoURL" placeholder="Expand to enter the repository details" />
```
Comment (fxiang1) [inline frontend/src/wizards/Argo/MultipleSourcesSelector.tsx:95]: Do we need to translate this message?

File: frontend/src/wizards/Argo/MultipleSourcesSelector.tsx [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+        </Fragment>
+      }
+    >
+      <WizHidden hidden={(data) => data.repositoryType}>
+        <Title headingLevel="h6">{t('Repository type')}</Title>
+      </WizHidden>
+      <WizTiles path="repositoryType">
+        <Tile id="git" value="git" label="Git" icon={<GitAltIcon />} description="Use a Git repository" />
```
Comment (fxiang1) [inline frontend/src/wizards/Argo/MultipleSourcesSelector.tsx:103]: Does the description need to be translated?

File: frontend/src/wizards/Argo/MultipleSourcesSelector.tsx [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+        <Tile id="helm" value="helm" label="Helm" icon={<HelmIcon />} description="Use a Helm repository" />
```
Comment (fxiang1) [inline frontend/src/wizards/Argo/MultipleSourcesSelector.tsx:104]: Same for here, translation needed?

**Review by fxiang1 — CHANGES_REQUESTED** [review]

File: frontend/src/wizards/Argo/MultipleSourcesSelector.tsx (hunk 4 above)
Comment (chenz4027) [reply frontend/src/wizards/Argo/MultipleSourcesSelector.tsx:103]: Ahh good catch, thanks Feng! I'll add translation here.

**Review by fxiang1 — APPROVED** [review]
Thanks Maggie! Looks great!

**Pull request merged**